BedrockServer.cpp

```diff
@@ -293,8 +293,13 @@ void BedrockServer::sync(SData& args,
                 // We need to grab this before peekCommand (or wherever our transaction is started), to verify that
                 // no worker thread can commit in the middle of our transaction. We need our entire transaction to
                 // happen with no other commits to ensure that we can't get a conflict.
+                uint64_t beforeLock = STimeNow();
                 server._syncThreadCommitMutex.lock();
 
+                // It appears that this might be taking significantly longer with multi-write enabled, so we're adding
+                // explicit logging for it to check.
+                SINFO("[performance] Waited " << (STimeNow() - beforeLock) << "us for _syncThreadCommitMutex.");
```

Comment:
Can we somehow capture the time spent waiting for locks in our overall performance metrics such that we can analyze them at the end of the command, rather than just logging them right here?  Ideally we'd aggregate all our timers into a single place, log them all at once together, and pass them all back to the client.